### DISCUSSION TITLE
Problem with odoo 14.0 database

### DISCUSSION THREAD

**Discussion by dani-xtendoo** [Tecnativa/doodba#463] [Q&A]
Good afternoon,
We have an installation of odoo v14.0 made with doodba working in a production environment, with 3 databases installed (test, pre-production and production), the problem we are faced with is that when accessing one of the databases from a device (for example test) and to another database (for example production) from another device, odoo stays loading for a long time, and in the log we see how it tries to load a database and at the moment the other, like this for a long time , which makes it impossible to perform any action on one device or the other, is there a way to be able to use 2 different databases at the same time, fluently and without having to duplicate the odoo instance?
Thanks for the help
Greetings,
                 Daniel Dominguez

**Comment by pedrobaeza** [comment]
That's possible without doing nothing special, so I don't get the problem, but mixing pre-production, test and production is not a good idea, as they may have different working code, impacting depending the state of the DB. That's maybe the problem you are facing. Anyways, it's not a problem to up several Doodba instances. You don't get too much resource penalty doing that.

**Comment by joao-p-marques** [comment]
Maybe https://github.com/Tecnativa/doodba-copier-template/blob/main/docs/daily-usage.md#environments can help with that. 

You should separate the different environments in different folders in the production server, using the appropriate docker-compose files for each one.
